sample/src/main/java/org/smartregister/sample/MainActivity.java

```diff
@@ -168,64 +171,52 @@ public void onNothingSelected(AdapterView<?> parent) {
             e.printStackTrace();
         }
 
-
         toggle.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
             public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                 String keyAlias = "sample";
                 // Get or create a key with the Alias name sample or create one if id does not exist
-                if(cryptographicHelper.getKey(keyAlias)==null)
-                {
+                if (cryptographicHelper.getKey(keyAlias) == null) {
                     cryptographicHelper.generateKey(keyAlias);
-                    Timber.i("key with alias %s generated",keyAlias);
+                    Timber.i("key with alias %s generated", keyAlias);
                 }
 
                 if (isChecked) {
-
                     try {
                         // read the text.txt while it is in plain text and write to file
                         FileInputStream inputStream = openFileInput(filename);
                         byte[] inputBytes = new byte[inputStream.available()];
                         inputStream.read(inputBytes);
                         byte[] encryptedContents = CryptographicHelper.encrypt(inputBytes, keyAlias);
                         FileOutputStream fileOutputStream = openFileOutput(filename, Context.MODE_PRIVATE);
-                        Timber.i("encrypted stuff to write %S ",new String(encryptedContents));
+                        Timber.i("encrypted stuff to write %S ", new String(encryptedContents));
                         encDecTextView.setText(new String(encryptedContents));
                         fileOutputStream.write((encryptedContents));
                         fileOutputStream.flush();
-
-
                     } catch (IOException e) {
                         e.printStackTrace();
                     }
```

Comment:
Let's add finally blocks to close the resources for clean up. 

Alternatively for this since we are already have Source and Target compatibility configured to 1.8 we can use the `try-with-resources` approach to manage the resources